### ISSUE TITLE
Bug:  Canvas not Styling

### ISSUE THREAD

**Issue by Runehkt** [mayurankv/Obsidian-Code-Styler#232]
Canvas cards when using code blocks does not style correctly
## Describe the bug
When creating code blocks in cards while working in canvas mode the code blocks are defaulting to vanilla. Seems it is a cards issue since the style works just fine if you import a file

I don't know

### Steps to reproduce

Create a card, make a code block with plugin active, input code, deselect card.

No

````markdown

````

### Expected behaviour

Style showing up, after card editing is done. even when the code block is deselected.

### Current behaviour

Style of code-block returns to vanilla after deselect the coding block. (you stop editing the code inside the code block) 

### Environment

Debian,Linux
- Plugin Version: 
1.1.7
- Obsidian Version:  
1.5.8
- Platform:  
 Both
- OS:  
 Debian
- Theme:  
 Dark Mode
- CSS Snippets:  
 Not sure

Ps: I'm new to programming scene in general so i might be missing something obvious, sorry if that is the case.

### Screenshots

![image](https://github.com/mayurankv/Obsidian-Code-Styler/assets/146981083/c9d144ee-29b4-4adf-8972-71aed1fab935)
![image](https://github.com/mayurankv/Obsidian-Code-Styler/assets/146981083/d5d69e16-9b4d-4c55-8da9-d212eb913f01)

**Comment by mayurankv** [comment]
No, this is a very valid bug report! I don't know how soon I can fix this as I'm currently super busy with exams but they're done soon so will be trying to get quite a few changes done then! Will definitely look into this.

**Comment by manolo777** [comment]
## Extra information
In my Canva file, when I press on the block, the code seems to render correctly. However, when the cursor moves out of the block, it returns to vanilla.
### When the cursor is in the code block..
 
![image](https://github.com/mayurankv/Obsidian-Code-Styler/assets/99422428/ba184535-ac0e-46da-bdd3-be32fe9fab44)

### When the cursor is in outside Code block
 
![image](https://github.com/mayurankv/Obsidian-Code-Styler/assets/99422428/f2109bb1-03a3-4f0f-8594-1019f961def1)

**Comment by mayurankv** [comment]
Ok this is an issue with the postprocessor (reading mode instead of editing mode). I need to change it so it also selects code in canvas blocks. Will add to the worklist. So sorry for the long wait!
